docs/source/hyper-parameter-search.rst

```diff
@@ -248,16 +302,10 @@ to use post-estimation features like scoring or prediction, we recommend using
 .. ipython:: python
 
    from dask_ml.wrappers import ParallelPostFit
-
-   params = {'estimator__alpha': np.logspace(-2, 1, num=1000),
-             'estimator__l1_ratio': np.linspace(0, 1, num=1000),
-             'estimator__average': [True, False]}
-
-   model = ParallelPostFit(SGDClassifier(tol=1e-3,
-                                         penalty="elasticnet",
-                                         random_state=0))
-   search = IncrementalSearchCV(model, params, random_state=0)
-   search.fit(X, y, classes=[0, 1])
+   params = {'estimator__alpha': np.logspace(-2, 1, num=1000)}
+   model = ParallelPostFit(SGDClassifier(tol=1e-3, random_state=0))
+   search = IncrementalSearchCV(model, params, 9, random_state=0)
```

Comment:
Use keywords for the parameters. The first two are probably fine as positional, but everything else should be named.